backend/src/openarchiefbeheer/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -556,361 +780,477 @@ msgid "rejected"
+#, fuzzy, python-format
+#| msgid ""
+#| "Destruction list \"%(list_name)s\" was made final by the record manager "
+#| "%(record_manager)s who assigned it to the archivist %(archivist)s."
+msgid ""
+"The destruction of destruction list \"%(list_name)s\" was aborted by record "
+"manager %(record_manager)s with reason: \"%(comment)s\"."
+msgstr ""
+"Vernietigingslijst \"%(list_name)s\" is definitief gemaakt door de "
+"archiefbeheerder %(record_manager)s die het heeft toegewezen aan de "
+"archivaris %(archivist)s."
+
+#: openarchiefbeheer/destruction/tests/endpoints/test_co_reviewers.py:133
+msgid ""
+"User \"%(user)s\" with the role of \"\" has replaced all the co-reviewers of "
+"the list \"%(list_name)s\" with: %(added_co_reviewers)s."
+msgstr ""
+"Gebruiker \"%(user)s\" met de rol \"%(role)s\" heeft alle mede-beoordelaars "
+"van de lijst \"%(list_name)s\" vervangen door: %(added_co_reviewers)s."
+
+#: openarchiefbeheer/destruction/tests/endpoints/test_co_reviewers.py:146
+#: openarchiefbeheer/destruction/tests/endpoints/test_co_reviewers.py:261
+#: openarchiefbeheer/logging/templates/logging/destruction_list_co_reviewers_added.txt:17
+#, python-format
+msgid "They added the comment: %(comment)s."
+msgstr ""
+
+#: openarchiefbeheer/destruction/tests/endpoints/test_co_reviewers.py:153
+#: openarchiefbeheer/destruction/tests/endpoints/test_co_reviewers.py:268
+#: openarchiefbeheer/logging/templates/logging/destruction_list_co_reviewers_added.txt:7
+#, python-format
+msgid "They also removed these co-reviewers: %(removed_co_reviewers)s."
+msgstr ""
+
+#: openarchiefbeheer/destruction/tests/endpoints/test_co_reviewers.py:248
+#, fuzzy, python-format
+#| msgid ""
+#| "User \"%(user)s\" with the role of \"%(role)s\" has added these users as "
+#| "co-reviewers to the list \"%(list_name)s\": %(added_co_reviewers)s."
+msgid ""
+"User \"%(user)s\" with the role of \"\" has added these users as co-"
+"reviewers to the list \"%(list_name)s\": %(added_co_reviewers)s."
+msgstr ""
+"Gebruiker \"%(user)s\" met de rol \"%(role)s\" heeft de volgende gebruikers "
+"toegevoegd als mede-beoordelaars aan de lijst \"%(list_name)s\": "
+"%(added_co_reviewers)s."
+
 #: openarchiefbeheer/destruction/tests/endpoints/test_list_destruction.py:109
 msgid "This list is already planned to be destroyed on 08/01/2024."
 msgstr ""
 
-#: openarchiefbeheer/destruction/tests/endpoints/test_list_destruction.py:189
+#: openarchiefbeheer/destruction/tests/endpoints/test_reviewresponse.py:241
+#, fuzzy, python-format
+#| msgid "User \"%(user)s\" has reviewed the list \"%(list_name)s\"."
 msgid ""
-"This list contains cases with archiving date later than 08/01/2024, so the "
-"destruction cannot be planned yet."
+"User \"%(user)s\" with the role of \"\" has reassigned destruction list "
+"\"%(list_name)s\"."
 msgstr ""
+"Gebruiker \"%(user)s\" met de rol \"%(role)s\" heeft de vernietigingslijst "
+"\"%(list_name)s\" opnieuw toegewezen."
```

Comment:
Same about these entries. The translations of the tests should pickup the same translations as the templates :thinking: